### PR TITLE
PROPS IPAX - GIANTS Adapter

### PR DESCRIPTION
**Pull request by PROPSIPAX**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'giants',
  params: {
    'zoneId': '584072408' // id provided by PROPS IPAX
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: support@props.id
- [x] official adapter submission

### CODE REVIEW

File: modules/giantsBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+    requiredParams: { required: true },
+    minimumParams: { sizes: [{}] },
+  },
+  sponsoredBy: 'sponsored_by',
+};
+const SOURCE = 'pbjs';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['giants'],
```
Comment (snapwich) [inline modules/giantsBidAdapter.js]: This is not needed.  The adapter is already called the `giants` adapter, so this alias is a duplicate.

**Comment by PROPSIPAX** [comment]
We removed alias and added GDPR.
Please check for me, thanks.

**Comment by PROPSIPAX** [comment]
Please let me know what I need to do next so my adapter can be updated to the adapter list on prebid.org.

**Comment by snapwich** [comment]
@PROPSIPAX 

to get on the download page you need to submit a bidders md file as outlined here: https://github.com/prebid/prebid.github.io#the-downloads-page

**Pull request merged**